crates/db_views_actor/src/structs.rs

```diff
@@ -80,6 +80,7 @@ pub struct CommunityView {
   pub subscribed: SubscribedType,
   pub blocked: bool,
   pub counts: CommunityAggregates,
+  pub banned: bool,
```

Comment:
Could you change this to `banned_from_community` . I know its kinda obvious based on its context, but we've tried to reserve `banned` for site bans only.